src/main/java/backend/Logic.java

```diff
@@ -127,9 +127,9 @@ public void refresh() {
         });
     }
 
-    public CompletableFuture<Boolean> getIssueMetadata(String repoId, List<Integer> issues) {
+    public CompletableFuture<Boolean> getIssueMetadata(String repoId, Map<Integer, String> issues) {
```

Comment:
Ah, alright. I'll pass the TurboIssues around, but I think making TurboIssues immutable should be in a different PR as it touches on the labelpicker / undo functionality.